robot_descriptions/allegro_hand_mj_description.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Allegro Hand V3 MJCF description."""
+
+from os import getenv as _getenv
+from os import path as _path
+
+from ._cache import clone_to_cache as _clone_to_cache
+
+REPOSITORY_PATH: str = _clone_to_cache(
+    "mujoco_menagerie",
+    commit=_getenv("ROBOT_DESCRIPTION_COMMIT", None),
+)
+
+PACKAGE_PATH: str = _path.join(REPOSITORY_PATH, "wonik_allegro")
+
+MJCF_PATH_LEFT: str = _path.join(PACKAGE_PATH, "left_hand.xml")
```

Comment:
There should also be one default `MJCF_PATH`, which is the one the loader will look for by default. (We could add a keyword argument to tell the loader which variant to load: opening it in https://github.com/robot-descriptions/robot_descriptions.py/issues/62 so that we don't scope-creep this PR.)

It otherwise fails with this error:
```
2024-03-26T09:30:39.9788535Z Traceback (most recent call last):
2024-03-26T09:30:39.9798338Z   File "/home/runner/work/robot_descriptions.py/robot_descriptions.py/tests/loaders/test_mujoco.py", line 32, in test
2024-03-26T09:30:39.9799125Z     load_robot_description(description)
2024-03-26T09:30:39.9800959Z   File "/home/runner/work/robot_descriptions.py/robot_descriptions.py/.tox/loader-mujoco/lib/python3.8/site-packages/robot_descriptions/loaders/mujoco.py", line 35, in load_robot_description
2024-03-26T09:30:39.9802180Z     raise ValueError(f"{description_name} is not an MJCF description")
2024-03-26T09:30:39.9802782Z ValueError: allegro_hand_mj_description is not an MJCF description
```